### PR TITLE
[DOCS] Improve Backend layout documentation

### PR DESCRIPTION
**Pull request by linawolf**
In preparation for documenting https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/922

Releases: main, 12.4

### CODE REVIEW

File: Documentation/PageTsconfig/Mod/WebLayout/BackendLayout.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
+..  include:: /Includes.rst.txt
+
+..  index::
+    mod; web_layout
+    Modules; Page
+..  _backend-layouts:
```
Comment (froemken) [inline Documentation/PageTsconfig/Mod/WebLayout/BackendLayout.rst:6]: Can we add a little bit of space here?

File: Documentation/PageTsconfig/Mod/WebLayout/BackendLayout.rst [hunk 2]
```diff
@@ -0,0 +1,102 @@
+..  include:: /Includes.rst.txt
+
+..  index::
+    mod; web_layout
+    Modules; Page
```
Comment (linawolf) [inline Documentation/PageTsconfig/Mod/WebLayout/BackendLayout.rst:5]:
```suggestion
```

File: Documentation/PageTsconfig/Mod/WebLayout/BackendLayout.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/PageTsconfig/Mod/WebLayout/BackendLayout.rst:3]: I removed the index as it was only copy-pasted and has no real content. However I cannot see where space is or was missing?

Comment (froemken) [reply Documentation/PageTsconfig/Mod/WebLayout/BackendLayout.rst:3]: After removing the index it looks good again.

**Review by froemken — APPROVED** [review]

**Pull request merged**